src/MICore/LaunchOptions.cs

```diff
@@ -53,9 +53,14 @@ public enum LaunchCompleteCommand
         ExecContinue,
 
         /// <summary>
+        /// Starts the debugger in the stopped state for debugging a core dump.
+        /// </summary>
+        CoreDump,
```

Comment:
I'll remove the CoreDump enum value and just key off of the CoreDumpPath on the LocalLaunchOptions class.